src/Symfony/Bundle/FrameworkBundle/FrameworkBundle.php

```diff
@@ -74,7 +74,10 @@ public function build(ContainerBuilder $container)
         parent::build($container);
 
         $container->addCompilerPass(new RoutingResolverPass());
-        $container->addCompilerPass(new ProfilerPass());
+        if (class_exists(AddConsoleCommandPass::class)) {
+            $container->addCompilerPass(new AddConsoleCommandPass());
+        }
+        $this->addCompilerPassIfExists($container, ProfilerPass::class);
```

Comment:
You're saying that I should write 
```
if (class_exists(ProfilerPass::class)) {
    $container->addCompilerPass(new ProfilerPass());
}
```
and duplicate the code ? 
or just say somewhere that I took this part of the code from #21283 which has already been reviewed ?